ci/ci/pr.py

```diff
@@ -84,6 +84,19 @@ def try_new_build(source, target):
                         'name': 'batch-test-cloud-sql-config',
                         'readOnly': True
                     }
+                }, {
+                    'volume': {
+                        'name': 'gsa-key-n6hf9',
```

Comment:
feels like this should be read from an environment variable

Reply:
this volume will be unavailable in this PR, you need to stage a CI->batch dependency